### PR TITLE
Added information about multi-factor auth

### PR DESCRIPTION
**Pull request by meker12**
Updated the Dev and Getting Started Guides with a link to information
about other authentication methods, including multi-factor
authentication.

### CODE REVIEW

**Comment by stretchservers** [comment]
Can one of the admins verify this patch?

**Comment by stretchservers** [comment]
Can one of the admins verify this patch?

**Comment by cyli** [comment]
retest this please

**Comment by meker12** [comment]
Fixed merge conflict. Pushing mfa documentation updates to production.

**Comment by lvh** [comment]
retest this please

**Comment by cyli** [comment]
@meker12 In case you are getting a bunch of notifications for us commenting "retest this please" and wondering what that's about, it's just a phrase that triggers our CI system to run tests.  :)  So not directed at you.  Thank you for your contribution!

**Comment by stretchservers** [comment]
Can one of the admins verify this patch?

**Comment by stretchservers** [comment]
Can one of the admins verify this patch?

**Comment by stretchservers** [comment]
Can one of the admins verify this patch?

**Comment by cyli** [comment]
ok to test

File: docbook/src/docbkx/autoscale-gettingstarted.xml [hunk 1]
```diff
@@ -619,6 +619,76 @@ xmlns:html="http://www.w3.org/1999/xhtml" xml:id="autoscale-getting-started" xml
                     </listitem>
                 </itemizedlist>
             </para>
+<<<<<<< HEAD
```
Comment (cyli) [inline docbook/src/docbkx/autoscale-gettingstarted.xml]: This extra line was left in from resolving the merge conflict I think.  It's causing errors building the docs with maven.

**Comment by meker12** [comment]
I messed up my git world, so I'm closing this pull request and submitting a new one, with better results. I hope.

**Pull request closed without merging**